<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLineEdit" name="editMsg">
        <property name="text">
         <string>ABC123abc</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnSendMsg">
        <property name="text">
         <string>发送消息</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="editIP"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>端口</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QSpinBox" name="spinPort">
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>1200</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLineEdit" name="editFile"/>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QProgressBar" name="progressBarSend">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QLineEdit" name="editSaveDir">
        <property name="text">
         <string>/home/wangyu/桌面</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QProgressBar" name="progressBarRec">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnSelectFile">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QPushButton" name="btnSendFile">
        <property name="text">
         <string>发送文件</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btnSaveDir">
        <property name="text">
         <string>保存路径</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>监听地址</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="separator"/>
   <addaction name="acHostInfo"/>
   <addaction name="separator"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
  </action>
  <action name="actStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
   </property>
   <property name="text">
    <string>停止监听</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/212.bmp</normaloff>:/images/images/212.bmp</iconset>
   </property>
   <property name="text">
    <string>清空文本</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="acHostInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/828.bmp</normaloff>:/images/images/828.bmp</iconset>
   </property>
   <property name="text">
    <string>本机信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>Server</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
